pages/projects/life-journeys.js

```diff
@@ -0,0 +1,140 @@
+import { Layout } from "../../components/organisms/Layout";
+import Head from "next/head";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+import { CallToAction } from "../../components/molecules/CallToAction";
+
+function HTMList({ tag = "ul", content, listClassName, liClassName }) {
+  const parseList = (content) =>
+    content
+      .split("*") // split the string on asterisks
+      .filter((item) => item) // filter out empty strings
+      .map((item, index) => (
+        <li className={liClassName} key={index}>
+          {item.trim()}
+        </li>
+      ));
+
+  return tag === "ul" ? (
+    <ul className={listClassName}>{parseList(content)}</ul>
+  ) : (
+    <ol className={listClassName}>{parseList(content)}</ol>
+  );
+}
```

Comment:
Neat-o, I will probably have to separate this into a new file for my PR if yours is merged first. Any issues with this function or was is pretty straightforward to use?

Reply:
Was very straightforward to use, originally had like 9 lines in each translation file for the lists which was annoying, so was happy to use this solution